docs/contributing.md

```diff
@@ -0,0 +1,304 @@
+# Contributing to Elpis Cloud
```

Comment:
From a readibility standpoint may be worth adopting Title Case throughout the headings in this file.

Reply:
Fixed in latest revision.